assets/src/edit-story/panels/actions.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * NOTE: This is temporary panel for being able to remove elements.
+ * It will be removed and replaced by using keyboard "Delete"
+ * once the approach for keyboard events has been confirmed.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { Panel, Title } from './shared';
+
+const Delete = styled.a`
+	cursor: pointer;
+	color: ${ ( { theme } ) => theme.colors.action };
+
+	&:hover {
+		color: ${ ( { theme } ) => theme.colors.danger };
+	}
+`;
+
+function ActionsPanel( { deleteCurrentSelection } ) {
+	return (
+		<Panel>
+			<Title>
+				{ 'Actions' }
```

Comment:
Shouldn't this string be translable?

Reply:
Yes, it should. Currently, however, no strings are translatable with `amp` domain in the app so I was just consciously ignoring this for now until a PR to make everything translatable. Perhaps there's an issue missing for making everything translatable?

Reply:
I have created a follow up ticket regarding translations - https://github.com/ampproject/amp-wp/issues/3842